src/ansi-c/c_typecheck_expr.cpp

```diff
@@ -3897,3 +3904,349 @@ optionalt<symbol_exprt> c_typecheck_baset::typecheck_gcc_polymorphic_builtin(
 
   return {};
 }
+
+static symbolt result_symbol(
+  const irep_idt &identifier,
+  const typet &type,
+  const source_locationt &source_location,
+  symbol_tablet &symbol_table)
+{
+  symbolt symbol;
+  symbol.name = id2string(identifier) + "::1::result";
```

Comment:
Always pretending to belong to source line 1?

Reply:
No, the first block/scope (which truly is the case, although the exact number wouldn't matter much either as long as it is unique).